lib/upgrade-db.js

```diff
@@ -0,0 +1,64 @@
+var MeshServer = require('strong-mesh-models').meshServer;
+var MeshServiceManager = require('strong-mesh-models').ServiceManager;
+var SQLite3 = require('loopback-connector-sqlite3');
+var async = require('async');
+var debug = require('debug')('upgrade-db');
+var fs = require('fs');
+var path = require('path');
+
+function checkUpgradeDb(baseDir, memoryDbPath, dryRun, callback) {
+  var meshDbPath = path.join(baseDir, 'strong-mesh.db');
+  fs.stat(meshDbPath, function(err) {
+    debug('Ensuring %s doesn\'t exist', meshDbPath);
+    if (!err) {
+      debug('Error: %s exist', meshDbPath);
+      return callback(new Error('Database ' + meshDbPath + ' already exists'));
```

Comment:
I think this won't work the second time with https://github.com/strongloop/strong-pm/pull/299/files#r48295496, because the json and the .db will exist

also, should we delete the json after creating the .db?  it will become out of date, and could then later be used if install was done with --use-memory-db, and that would be kindof bad